pkg/registry/pod/etcd/etcd.go

```diff
@@ -142,7 +143,14 @@ func (r *BindingREST) New() runtime.Object {
 // Create ensures a pod is bound to a specific host.
 func (r *BindingREST) Create(ctx api.Context, obj runtime.Object) (out runtime.Object, err error) {
 	binding := obj.(*api.Binding)
-	err = r.assignPod(ctx, binding.PodID, binding.Host)
+	// TODO: move me to a binding strategy
+	if len(binding.Target.Kind) != 0 && (binding.Target.Kind != "Node" && binding.Target.Kind != "Minion") {
```

Comment:
We need to do this for host ports, out-of-disk, and other scenarios/resources, also.

@erictune @dchen1107 @rjnagal @vishh Is someone working on implementing this protocol?